WalletWasabi/BitcoinCore/RpcParser.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using NBitcoin;
+using NBitcoin.RPC;
+using Newtonsoft.Json.Linq;
+using WalletWasabi.BitcoinCore.RpcModels;
+
+namespace WalletWasabi.BitcoinCore
+{
+	public class RpcParser
+	{
+		public static RpcPubkeyType ConvertPubkeyType(string pubKeyType)
```

Comment:
Why is this necessary? Or, how is this better than using the `NBitcoin.Script.IsScriptType()`?

Reply:
Faster. If the data is already available why try to do a bunch of computations to figure that out.

Reply:
I am pretty sure it is not faster, in fact i would bet it is slower (impossible to be sure without meassuring it). More elegant, yes.

Reply:
This are really very negligible improvements, as we agreed if it turns out to be a perf issue we fix it later.

Reply:
You can look into the code of `IsScriptType`.

Reply:
I did, that's why I say that this has to be slower.